_sass/_blog.scss

```diff
@@ -0,0 +1,11 @@
+.blog-embed {
+	// position: relative;
+	text-align: center;
+
+	iframe {
+		width: 100%;
+		height: 230px;
```

Comment:
I don't think using `px` widths here is a good idea.

Reply:
I prefer using `rem` but on hiDPI displays, browsers will scale `px` as needed anyway. The whole "using px units breaks hiDPI" support isn't really true due to how browsers handle px as DPI-independent pixels nowadays (by adjusting the default zoom behind the scenes).